File: app/src/main/java/com/ragazm/notepad/recycler/ItemClickListener.java
package com.ragazm.notepad.recycler;

import android.view.View;

/**
 * Created by Andris on 022 22.10.17.
 * Item click listener
 */



public interface ItemClickListener {
    void onItemClick(View view, int position);
    void onItemPress(View view, int position);

}

File: app/src/main/java/com/ragazm/notepad/recycler/ItemClickListener.kt
package com.ragazm.notepad.recycler

import android.view.View

/**
 * Created by Andris on 022 22.10.17.
 * Item click listener
 */


interface ItemClickListener {
    fun onItemClick(view: View, position: Int)
    fun onItemPress(view: View, position: Int)

}
